book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @auther = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    rental = Rental.new(date, person, self)
+    @rentals << rental unless @rentals.include?(rental)
```

Comment:
- [ ] I believe this line of code which tries to add the newly created `rental` to the `@rentals` array is no longer necessary as that operation has been taken care of already in the `Rental` class constructor like this 👇 
```ruby
book.rentals << self unless book.rentals.include?(self)
```